server/src/main/java/com/cloud/api/ApiServer.java

```diff
@@ -1011,7 +1011,7 @@ public boolean verifyRequest(final Map<String, Object[]> requestParameters, fina
 
             // if api/secret key are passed to the parameters
             if ((signature == null) || (apiKey == null)) {
-                logger.debug("Expired session, missing signature, or missing apiKey -- ignoring request. Signature: " + signature + ", apiKey: " + apiKey);
+                logger.debug("Expired session, missing signature, or missing apiKey -- ignoring request. Signature: {}, apiKey: {}", signature, apiKey);
```

Comment:
should we consider this one to be of info level?

Reply:
If we are changing this, maybe we should set it as warn?